ssl/s3_lib.c

```diff
@@ -4796,7 +4796,7 @@ EVP_PKEY *ssl_generate_pkey_group(SSL *s, uint16_t id)
 /*
  * Generate parameters from a group ID
  */
-EVP_PKEY *ssl_generate_param_group(uint16_t id)
+EVP_PKEY *ssl_generate_param_group(SSL *s, uint16_t id)
```

Comment:
Where are you using this?

Reply:
It isn't being used yet, but it will be necessary when `EVP_PKEY_CTX_new_id` is changed over.  See the TODO comment about 20 lines later.

I started on the conversion but soon realised that it isn't possible to complete yet.  While undoing the relevant changes, I figured this was okay to stay since it will be required.

Reply:
Okie.  Thanks